DEPLOYMENT_GUIDE.md

```diff
@@ -0,0 +1,57 @@
+## Deployment Guide
+
```

Comment:
one liner for what the guide is and what someone would get out of it

This guide walks through the process of deploying contracts to public Ethereum networks and verifying contracts on Etherscan in the individual sub-repos of the airswap-protocols. The steps require one to have an Ethereum address mneumonic, Etherescan API key, and Infura key. All of these are free to get.